students/DanCornutt/lessons/lesson07/tests/test_database.py

```diff
@@ -0,0 +1,9 @@
+"""This is the test script for basic operations"""
+from unittest import TestCase
+from unittest.mock import MagicMock, patch
+import pytest
+from database import main, import_data, add_collection_csv, show_rentals, \
+    show_available_products, MongoDBConnection
+
+
+# Need to set up fixtures
```

Comment:
Well, you thought about tests. ;-)

Reply:
Yes, fixtures and mocks are on my list to come back to. Things I know I need but still a bit foggy on implementation. I'll get it before the class it up!